shapash/webapp/smart_app.py

```diff
@@ -2119,17 +2119,35 @@ def update_prediction_picking(feature,
             else:
                 raise PreventUpdate
 
-            self.components['graph']['prediction_picking'].figure = self.explainer.plot.scatter_plot_prediction(
-                selection=self.subset,
-                max_points=points,
-                label=self.label
-            )
+            if self.explainer.y_target is not None:
+                self.components['graph']['prediction_picking'].figure = self.explainer.plot.scatter_plot_prediction(
+                    selection=self.subset,
+                    max_points=points,
+                    label=self.label
+                )
 
-            self.components['graph']['prediction_picking'].figure['layout'].clickmode = 'event+select'
-            # Adjust graph with adding x and y axis titles
-            self.components['graph']['prediction_picking'].adjust_graph(
-                x_ax="Target",
-                y_ax="Prediction")
+                self.components['graph']['prediction_picking'].figure['layout'].clickmode = 'event+select'
+                self.components['graph']['prediction_picking'].adjust_graph(
```

Comment:
Ici je veux bien que tu remettes le commentaire 
# Adjust graph with adding x and y axis titles